react/features/base/conference/middleware.web.js

```diff
@@ -61,12 +68,33 @@ MiddlewareRegistry.register(store => next => action => {
 
         break;
     }
-    case TOGGLE_SCREENSHARING: {
-        getMultipleVideoSendingSupportFeatureFlag(getState()) && _toggleScreenSharing(action, store);
+    case TOGGLE_SCREENSHARING:
+        if (typeof APP === 'object') {
```

Comment:
You no longer need this check since this is a web-only middleware. Which brings in the quesiton: doesn't this now break mobile because this logic won't be applied?

Reply:
No, the code that was moved was only executed on web. So no change on mobile

Reply:
👍